libs/utils/analysis/latency_analysis.py

```diff
@@ -387,9 +387,11 @@ def _taskState(self, state):
             256: "W", # TASK_WAKING
             512: "P", # TASK_PARKED
            1024: "N", # TASK_NOLOAD
-           2048: "n", # TASK_NEW
         }
-        TASK_MAX_STATE = 4096
+        kver = float(self._trace.platform.get('kernel_version', '3.18'))
+        if kver > 4.7:
```

Comment:
I didn't add the `__cmp__` in the end. However this needs to change anyway because 'kernel_version' isn't a field any more. I put a couple of suggested fixup patches here: https://github.com/bjackman/lisa/commits/fix-tasks-states